render-utils.js

```diff
@@ -1,12 +1,16 @@
 export function renderPoll(question, optionA, optionB, votesA, votesB) {
     const div = document.createElement('div');
-    
+    const optionAElDiv = document.createElement('p');
+    const optionBElDiv = document.createElement('p');
     let optionQuestion = question;
```

Comment:
Since this doesn't get mutated it would be better to declare this as `const`

Reply:
roger that